packages/mask/src/plugins/Trader/pipes.ts

```diff
@@ -40,7 +40,10 @@ export const resolveTradeProviderName = createLookupTableResolver<TradeProvider,
         [TradeProvider.PANGOLIN]: 'PangolinDex',
         [TradeProvider.TRISOLARIS]: 'Trisolaris',
         [TradeProvider.WANNASWAP]: 'WannaSwap',
+        [TradeProvider.VENOMSWAP]: 'VenomSwap',
+        [TradeProvider.OPENSWAP]: 'OpenSwap',
         [TradeProvider.MDEX]: 'Mdex',
+        [TradeProvider.DEFIKINGDOMS]: 'DefiKingDoms',
```

Comment:
```suggestion
        [TradeProvider.DEFIKINGDOMS]: 'DefiKingdoms',
```

`Kingdom` is a word.